### ISSUE TITLE
Add landmark regions/roles to manager interface (accessibility defect)

### ISSUE THREAD

**Issue by paulbohman** [modxcms/revolution#12149]
For compliance with Web Content Accessibility Guidelines (WCAG)

There are no landmark regions or roles specified. Landmark regions help screen reader users navigate quickly to regions without having to read through the whole page. Here are the regions I would suggest for the main manager layout:
<pre><code>
&lt;header role=&quot;banner&quot;&gt; <br />
&lt;div role=&quot;search&quot;&gt;&lt;/div&gt;<br />
&lt;nav role=&quot;navigation&quot; aria-label=&quot;manager menu&quot;&gt; &lt;!--(for the menu on the left)--&gt; &lt;/nav&gt;<br />
&lt;nav role=&quot;navigation&quot; aria-label=&quot;user and system menu&quot;&gt; &lt;!--(for the menu on the right)--&gt; &lt;/nav&gt; <br />
&lt;/header&gt;<br />
&lt;ul role=&quot;tablist&quot; aria-label=&quot;Tree view tablist&quot;&gt; &lt;!--(or come up with a better label for this)--&gt;<br />
&lt;li role=&quot;tab&quot;&gt;Resources&lt;li&gt;&lt;li role=&quot;tab&quot;&gt;Elements&lt;/li&gt;&lt;li role=&quot;Files&quot;&gt;<br />
&lt;/ul&gt; <br />
&lt;div role=&quot;tabpanel&quot;&gt; <br />
&lt;nav role=&quot;navigation&quot; aria-label=&quot;Resource Tree&quot;&gt; &lt;ul&gt;...&lt;/ul&gt;&lt;/nav&gt;<br />
&lt;nav role=&quot;navigation&quot; aria-label=&quot;Element Tree&quot;&gt; &lt;ul&gt;...&lt;/ul&gt;&lt;/nav&gt;<br />
&lt;nav role=&quot;navigation&quot; aria-label=&quot;File Tree&quot;&gt; &lt;ul&gt;...&lt;/ul&gt;&lt;/nav&gt;<br />
&lt;/div&gt;<br />
&lt;main role=&quot;main&quot;&gt;<br />
All of the main content goes here (page edits, snippet edits, dashboard content, etc.)<br />
&lt;/main&gt;<br />
&lt;footer role=&quot;contentinfo&quot;&gt;<br />
(Right now MODx manager does not have a footer, but this is how you would mark it up)<br />
&lt;/footer&gt;
</code></pre>

Note: You could also achieve the same functional result using only &lt;div&gt; tags with role attributes instead of the HTML 5 tags plus the role attributes, but the latter solution is more forward-thinking.

See accessibility to-do list at https://dequeuniversity.com/modx-accessibility.html#overall-structure

**Comment by rthrash** [comment]
This is being addressed at the [MODX Accessible CMS Project](https://github.com/modxcms/a11y/issues/), and with Paul's agreement has been moved there. For more details, see https://github.com/modxcms/revolution/issues/12156#issuecomment-72382480. Thank you!
